pyprep/utils.py

```diff
@@ -64,3 +64,29 @@ def filter_design(N_order, amp, freq):
     )
     kernel = np.multiply(kernel[0 : N_order + 1], (np.transpose(hamming_window[:])))
     return kernel
+
+
+def SplitList(mylist, chunk_size):
+    """Split list in chunks.
+
+    Parameters
```

Comment:
```suggestion
    """Split list in chunks.

    See: https://stackoverflow.com/a/312466/5201771

    Parameters
```

Small code snippets like this one dont have to be cited with name and everything. A link to the stackoverflow post should be enough.

You can generate a better link (like I did for you now) by clicking on "share" as in the left lower corner of this screenshot:

![image](https://user-images.githubusercontent.com/9084751/86565953-156a9800-bf69-11ea-8e5d-daa8bdd92c94.png)